src/flex-example-app-android.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<application xmlns="http://ns.adobe.com/air/application/19.0">
+    <id>com.example.android.snake</id>
```

Comment:
but we also want them to connect it to their account so that they can control their account from our website. they won't have access to the dashboard site for our own sample apps